autoreporter_addons/sqlmap/reporter.py

```diff
@@ -65,11 +67,15 @@ def get_email_template_fragments() -> List[ReportEmailTemplateFragment]:
     @staticmethod
     def get_normal_form_rules() -> Dict[ReportType, Callable[[Report], NormalForm]]:
         """See the docstring in the Reporter class."""
-        return {
-            SQLmapReporter.SQL_INJECTION: lambda report: Reporter.dict_to_tuple(
+
+        def rule(report: Report) -> NormalForm:
+            hostname = urllib.parse.urlparse(report.target).hostname
+
+            return Reporter.dict_to_tuple(
                 {
                     "type": report.report_type,
-                    "target": get_url_normal_form(report.target),
+                    "target": get_domain_normal_form(hostname) if is_domain(hostname) else hostname,
```

Comment:
jest to generalnie w paru miejscach w kodzie artemisa, moze po prostu zrobic to jako warunek returna tej funkcji w main source?

Reply:
ja to widze tylko w tym miejscu a więc yagni